dynet/nodes-conv2d.cc

```diff
@@ -70,6 +70,26 @@ Dim Conv2D::dim_forward(const vector<Dim>& xs) const {
   return Dim(output_shape, bs);
 }
 
+int Conv2D::autobatch_sig(const ComputationGraph & cg, SigMap &sm) const {
+  Sig s(nt::conv2d);
+  if(dim.bd == 1) {
```

Comment:
Note that this will not batch convolutions where the filter has batch dimension 1, but the input has batch dimension more than one. This seems like a case that we ideally should support, so could you either support this, or add a TODO here noting that this isn't supported yet, but should be?